newrelic-python.ini

```diff
@@ -45,12 +48,14 @@ app_name = your_app_name
 # NEW_RELIC_MONITOR_MODE environment variable.
 monitor_mode = true
 
-# When enabled, the agent's security module is not loaded
-# this would mean changes on soft reloads won't work
-security.force_complete_disable = false
-# When enabled, the agent collects security data about your
-# application and reports this data to the New Relic.
-security.enable = true
+# To completely disable security set flag to false If the flag is
+# set to false, the security module is not loaded. This property
+# is read only once at application start.
+security.agent.enable = agent_enabled_flag
```

Comment:
It should be security.agent.enabled = agent_enabled_flag

Reply:
Updated